### DISCUSSION TITLE
Dashboard: link to other page in text

### DISCUSSION THREAD

**Discussion by XanderC137** [quarto-dev/quarto-cli#7690] [Q&A] [answered]
### Description

As the title says, I would want to have a clickable hyperlink in my text that brings users to another page. I can't really find documentation on something like this so **if** it's possible, how would I go about this?

**Comment by mcanouil** [comment]
Did you try using regular markdown syntax as described in the markdown basics page of the documentation?

If that is not what you please elaborate and possibly use/share a concrete example.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by XanderC137** [reply]
I have a dashboard looking like this:

<img width="1465" alt="Scherm­afbeelding 2023-11-24 om 13 44 52" src="https://github.com/quarto-dev/quarto-cli/assets/151828460/58dfb4d8-5795-4d86-af7d-354f92c8fdb4">

At the bottom in the first card I want the word "datatable" to link to the page "data" in this dashboard. I don't see how to use the markdown for links "\<https://quarto.org>" to refer to a page within the dashboard.

**Reply by mcanouil** [reply]
> to another page

By this you meant marking internal links to dashboard components.
Now I understand.
Unfortunately, a screenshot is not a reproducible example so I can't use it to provide detailed/accurate help.

**Reply by XanderC137** [reply]
Right now my code just looks something like this:

````qmd
---
title: "<b>Identified cryostock isolates</b>"
format: dashboard
scrolling: true
---
# Visualisation
<code for plotly plots>

# Data
``` {r}
DT::datatable(df)
```

# Info
::: {.card title="Interactive figures"}
Text and table
:::

::: {.card title="Interactive figures"}
Text and tables
:::
````

What I would very much like is a way to link to the "# Data" page in my dashboard within the text in one of my cards on the last page.

**Reply by mcanouil** [reply, marked as answer]
As I said, it's just regular markdown.

````qmd
---
title: "**Identified cryostock isolates**"
format: dashboard
scrolling: true
---

# Visualisation

[Link to data page](#data)

# Data

```{r}
knitr::kable(head(mtcars))
```

# Info

::: {.card title="Interactive figures"}
[Link to visualisation page](#visualisation)

Text and table
:::

::: {.card title="Interactive figures"}
Text and tables
:::
````

**Reply by XanderC137** [reply]
That did the trick! Thank you for your help with this pretty simple problem...I just couldn't find out the part where "#data" serves as a link. Now it works though :))
